ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/LedgerApiTestTool.scala

```diff
@@ -39,16 +41,38 @@ object LedgerApiTestTool {
   private def exitCode(summaries: Vector[LedgerTestSummary], expectFailure: Boolean): Int =
     if (summaries.exists(_.result.isLeft) == expectFailure) 0 else 1
 
-  private def printAvailableTests(): Unit = {
-    println("Tests marked with * are run by default.")
+  private def printAvailableTests(config: Config): Unit = {
```

Comment:
Further point: this list can become quite long and I'm not sure this is necessarily what you want every time. What do you think about having `--list` that behaves as before and (e.g.) `--list-verbose` to list each test case in each suite?

Reply:
As the test suite titles are not the test suite names used for exclusion at the moment, printing only the test suite names seems a bit misleading.